.github/workflows/build_azlinux_jupyter_scipy_image.yml

```diff
@@ -0,0 +1,22 @@
+name: Build Push to Quay
```

Comment:
```suggestion
name: Build and push azlinux-jupyter-scipy
```
I found out the hard way that it gets very confusing when all the workflows have a similar name:
<img width="307" alt="Screenshot 2023-09-29 at 1 28 38 PM" src="https://github.com/uc-cdis/containers/assets/4224001/ef4f53d9-53d8-4c28-b006-649d8ea413d2">